### ISSUE TITLE
Does the training set include MLT 17 validation set?

### ISSUE THREAD

**Issue by TMYsecret** [facebookresearch/MultiplexedOCR#19]
Hi, thank u for sharing the MultiplexedOCR codes. 
I noticed that the comments in ./multiplexer/data/catalog.py metioned 'It's a common practice in academia to include mlt17 val into training  as there's no validation benchmark'. I wonder if the training set of MultiplexedOCR contains MLT 17 validation (1800 images in total)?
look forward to ur reply~
thx~
